### PR TITLE
select poster image as title instead of select primary image

### PR DESCRIPTION
**Pull request by bhsiao55**
* Add `modalTitle` prop to image asset picker modal and image asset picker field.

### CODE REVIEW

File: src/plugins/dam/components/image-asset-picker-modal/index.jsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ class ImageAssetPickerModal extends React.Component {
     galleryNode: PropTypes.instanceOf(Message),
     isOpen: PropTypes.bool,
     label: PropTypes.string,
+    modalTitle: PropTypes.string,
```
Comment (hunyoboy) [inline src/plugins/dam/components/image-asset-picker-modal/index.jsx:32]: @bhsiao55 since you're already in the modal, you can just call this title.

**Review by hunyoboy — COMMENTED** [review]
@bhsiao55 looks good. just a comment.

File: src/plugins/dam/components/image-asset-picker-modal/index.jsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ class ImageAssetPickerModal extends React.Component {
     galleryNode: PropTypes.instanceOf(Message),
     isOpen: PropTypes.bool,
     label: PropTypes.string,
```
Comment (hunyoboy) [inline src/plugins/dam/components/image-asset-picker-modal/index.jsx:31]: @bhsiao55 i don't think you need the label props now really! you can just default your 'title' to current label's value.

File: src/plugins/dam/components/image-asset-picker-modal/index.jsx (hunk 1 above)
Comment (bhsiao55) [reply src/plugins/dam/components/image-asset-picker-modal/index.jsx]: yup will change

File: src/plugins/dam/components/image-asset-picker-modal/index.jsx (hunk 2 above)
Comment (bhsiao55) [reply src/plugins/dam/components/image-asset-picker-modal/index.jsx:31]: label still being used by other components

**Review by hunyoboy — APPROVED** [review]

**Pull request merged**